src/contexts/environment-context.js

```diff
@@ -64,6 +64,9 @@ export const EnvironmentProvider = ({ children }) => {
       url: `${relativeHost}/static/frontend/env.json`
     })
     let context = response.data;
+
+    // split the comma-separated string which tells ui the support section to hide
+    context.hidden_support_sections = context.hidden_support_sections.split(',')
```

Comment:
This is definitely the easier approach

Reply:
yep, I will update the ui-chart pr shortly.